File: beecrowd/javascript/bee1036.js
var input = require("fs").readFileSync("stdin", "utf8");

const a = parseFloat(input.split(" ")[0]);
const b = parseFloat(input.split(" ")[1]);
const c = parseFloat(input.split(" ")[2]);

const delta = Math.pow(b, 2) - 4 * a * c;

if (delta < 0 || a === 0) {
    console.log("Impossivel calcular");
} else {
    const r1 = (-b + Math.sqrt(delta)) / (2*a);
    const r2 = (-b - Math.sqrt(delta)) / (2*a);

    console.log("R1 = " + r1.toFixed(5));
    console.log("R2 = " + r2.toFixed(5));
}

File: python/bee1036.py
from math import sqrt

a, b, c = [float(x) for x in input().split(' ')]

try:
    delta = b*b - 4*a*c
    r1 = (-b + sqrt(delta)) / (2*a)
    r2 = (-b - sqrt(delta)) / (2*a)

    print(f'R1 = {r1:.5f}')
    print(f'R2 = {r2:.5f}')
except:
    print('Impossivel calcular')
